app/scripts/views/permissions.js

```diff
@@ -22,6 +63,8 @@ define(function (require, exports, module) {
       this._account = this.user.initAccount(this.model.get('account'));
 
       this.type = options.type;
+
+      this._validatePermissions(this.relier.get('permissions') || []);
```

Comment:
I think ideally this will be done in the oauth relier, as part of fixing #2508. For a future PR.